src/components/Reports/PeopleReport/PeopleReport.jsx

```diff
@@ -505,25 +510,22 @@ class PeopleReport extends Component {
     );
 
     const onChangeBioPosted = async (bio) => {
-      const userId = this.state.userId || this.props.match?.params?.userId;
```

Comment:
I have a concern regarding the current userId retrieval method, as we have made changes in this area. While I believe your proposed method is sound, I would like to confirm with @juneluo to ensure that changing this line is acceptable.

Reply:
The change looks good to me because previously `this.state.userId` was set in `componentDidMount` and using the same `userId` that `this.props.userProfile` uses as the param of the `getUserProfile`. So I think this change will work as intended.